docs/Roadmap.md

```diff
@@ -0,0 +1,112 @@
+# EaseProbe Roadmap
+EaseProbe plans to align closely with the Cloud Native ecosystem in providing service probes and notifications.
+
+- [EaseProbe Roadmap](#easeprobe-roadmap)
+  - [Product Principles](#product-principles)
+  - [Features](#features)
+    - [Probe specific](#probe-specific)
+    - [Notify specific](#notify-specific)
+  - [Roadmap 2022](#roadmap-2022)
+    - [General](#general)
+    - [Probes](#probes)
+    - [Notify](#notify)
+
+
+## Product Principles
+EaseProbe tries to do two things but do them extremely well - **Probe** and **Notify**.
+
+1. **Probe and Notify**. Aims to provide the ability to easily **probe** servers, services and API endpoints reliably and **notify** on status updates.
```

Comment:
I'd like the first principle is lightweight, standalone, and KISS.

Reply:
Added please have a look